docs/api-references/counts.md

```diff
@@ -0,0 +1,29 @@
+## Counts - API
+
+### `GET` List Count Value
+```
+/api/counts
```

Comment:
URI should be `api/statistics/counts`. And when you implement, create new Controller to handle this `StatisticController`